File: Starter Problems 2/rain.py
op = open('rainin.txt', 'r')
n, c = map(int, op.readline().split())
total = 0
count = 0
for i in range(n):
	total += int(op.readline())
	count += 1
	if total >= c:
		ot = open('rainout.txt', 'w')
		ot.write(str(count))
		break

File: Starter Problems 2/rain.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n,c;
    ifstream inFile;
    inFile.open("rainin.txt");
    inFile >> n;
    inFile >> c;
    int nums[n];
    for (int i = 0;i<n;i++) {
        inFile >> nums[i];
    }
    int days = 1;
    int total = 0;
    for (int i=0;i<n;i++) {
        total += nums[i];
        //printf("%d %d %d\n",total,nums[i],days);
        if (total < c) {
            days += 1;
        }
    }

    inFile.close();
    ofstream outFile;
    outFile.open("rainout.txt");
    outFile << days;
    outFile.close();
}
